Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -250,6 +236,29 @@ namespace O3DE::ProjectManager
         StopPython();
     }
 
+    void PythonBindings::OnStdOut(const char* msg)
+    {
+        AZStd::string message{ msg };
+        // escape % characters by using %% in the format string to avoid crashing
+        AZ::StringFunc::Replace(message, "%", "%%", true /* case sensitive since it is faster */);
+        AZ_TracePrintf("Python", message.c_str());
```

Comment:
I did consider it, but wasn't sure it would be a good idea to bypass the `TraceMessageBus::OnPrintf` output since `TraceMessageBus::OnOutput` gets output from warnings and errors as well.  Honestly, I could go either way though, no strong preference.

Reply:
Well I would prefer to `AZ::Debug::Trace::Output` to avoid copying of the message.

Also the code you have is in ProjectManager, why would it matter if the TransMessageBus::OnPrintf handler is skipped when the Project Manager itself controls whether it listens to it at all

Reply:
Should be addressed in #10168